exporting/exporting.conf

```diff
@@ -29,6 +29,7 @@
     # send names instead of ids = yes
     # send charts matching = *
     # send hosts matching = localhost *
+    # send variables = no
```

Comment:
This should be in the remote write section.

Reply:
yes, I will change it. or after every  modules have send variables func, it will be added in every section.

Reply:
Please don't forget to move the option to the correct place.